### PR TITLE
[network-topology] Add support for Django 2.2 #36

### LINKED ISSUE #36: [network-topology] Add support for django 2.2
Add support for django 2.2, but first we need to upgrade django-netjsongraph to support django 2.2: https://github.com/netjson/django-netjsongraph/issues/93.

### PR DESCRIPTION
**Pull request by IAmAMissingColon**
Enabled Django2.2

Fixes #36

### CODE REVIEW

**Comment by IAmAMissingColon** [comment]
Here's the GCI link 
* https://codein.withgoogle.com/dashboard/task-instances/4716687773925376/

**Comment by IAmAMissingColon** [comment]
> Any reason you chose to change the lower bound from 2.1 to 2.2?

Because of travis actually.. I can change it back if you suggest :smile:

**Comment by IAmAMissingColon** [comment]
@atb00ker 
* Commitng the changes you suggested please wait :smile: 
* Agree with the second point will keep it in mind :smile:

**Comment by IAmAMissingColon** [comment]
@atb00ker 

* Didn't get what is exactly mean by exclude means completely remove it ?

**Comment by IAmAMissingColon** [comment]
Screenshot_20191204_142121

* Something like that ?, sorry if it's worng I will correct it ! :smile:

**Comment by IAmAMissingColon** [comment]
I will push the changes if it's correct :smile:

**Comment by IAmAMissingColon** [comment]
`matrix:
  exclude:
   - python: "2.7"
     env: DJANGO="django>=2.1,<2.2"
   - python: "2.7"
     env: DJANGO="django>=2.1,<2.3"`

**Comment by IAmAMissingColon** [comment]
Sorry... This is correct I Think 

Screenshot_20191204_144400

**Comment by IAmAMissingColon** [comment]
Build has stopped :disappointed:

**Comment by IAmAMissingColon** [comment]
seems that python2.7 with django is failing

File: .travis.yml [hunk 1]
```diff
@@ -7,13 +7,16 @@ python:
   - "2.7"
 
 env:
-  - DJANGO="django>=1.11,<1.12"
```
Comment (2stacks) [inline .travis.yml]: Removing this is causing python2.7 to fail.

File: .travis.yml [hunk 2]
```diff
@@ -7,13 +7,16 @@ python:
...
   - DJANGO="django>=2.1,<2.2"
+  - DJANGO="django>=2.2,<2.3"
 
 matrix:
   exclude:
    - python: "2.7"
      env: DJANGO="django>=2.1,<2.2"
+  exclude:
```
Comment (2stacks) [inline .travis.yml]: You shouldn't have to duplicate the `exclude` block.

File: .travis.yml [hunk 3]
```diff
@@ -6,14 +6,20 @@ python:
   - "3.5"
   - "2.7"
 
-env:
+env:
```
Comment (2stacks) [inline .travis.yml]: Be careful with unnecessary white space.

**Comment by IAmAMissingColon** [comment]
> I think that should do it. I believe removing the `sudo` section can be done in another task.

I can do it :heart_eyes: :-)

**Comment by atb00ker** [comment]
> I can do it heart_eyes :-)

Please don't do it here, that's a different task that you need to claim to do it! :smile:

**Comment by IAmAMissingColon** [comment]
Okay, Thank You Sir :smiley:

**Pull request merged**